docs/source/articles/run-tests.html.md

```diff
@@ -0,0 +1,203 @@
+---
+title: Run Tests
```

Comment:
I'm not sure where this should be, but it would be good to offer suggestions about setting up builds which is mostly running tests. Like something about parallelizing core/admin/storefront/plugins or some such to improve build times.

Reply:
I recommend writing a how to doc for setting up CI, and reference the testing docs as necessary from there.